js/jquery.socialfeed.js

```diff
@@ -720,8 +720,7 @@ if (typeof Object.create !== 'function') {
 
                 getData: function(url) {
                     var limit = '&num='+ options.rss.limit,
-                      request_url = Feed.rss.api + limit + '&q=' + encodeURIComponent(url);
-
+                    request_url = Feed.rss.api + limit + '&q=' + encodeURIComponent(url);
```

Comment:
And here should be an indent. It should align with limit since it's in the same var declaration.